src/markdown/parser.cr

```diff
@@ -92,7 +100,12 @@ class Markdown::Parser
         break
       end
 
-      if starts_with_star?(line) || starts_with_backticks?(line) || starts_with_digits_dot?(line)
+      if (
+        starts_with_bullet_list_marker?(line, '*') ||
+        starts_with_bullet_list_marker?(line, '+') ||
+        starts_with_bullet_list_marker?(line, '-') ||
```

Comment:
This makes me wonder if the prefix argument should be optional and it should check for any of the three then. Basically `prefix ? str[pos].chr == prefix : (str[pos].chr == '*' || str[pos].chr == '-' || str[pos].chr == '+')`, maybe assigning `str[pos].chr` to a local beforehand.

What do you think?